Declarations/Constants.ts

```diff
@@ -141,4 +141,45 @@ export const DependencyTypeName = {
     InProc: "InProc",
 }
 
-export const HeartBeatMetricName = "HeartBeat";
\ No newline at end of file
+export const HeartBeatMetricName = "HeartBeat";
+
+export const StatsBeatMetricName = "StatsBeat";
+
+export const StatsBeatResourceProvider = {
+    appsvc: "appsvc",
+    function: "function",
+    vm: "vm",
+    unknown: "unknown",
+}
+
+export const StatsBeatAttach = {
+    codeless: "codeless",
+    sdk: "sdk",
+}
+
+export const StatsBeatCounter = {
+    REQUEST_SUCCESS: "Request Success Count",
+    REQUEST_FAILURE: "Requests Failure Count",
+    REQUEST_DURATION: "Request Duration",
+    RETRY_COUNT: "Retry Count",
+    THROTTLE_COUNT: "Throttle Count ",
+    EXCEPTION_COUNT: "Exception Count",
+}
+
+export enum StatsBeatFeature {
+    NONE = 0,
+    DISK_RETRY = 1,
+    AAD_HANDLING = 2,
+}
+
+export enum StatsBeatInstrumentation {
```

Comment:
i would like to see how you can decode the long back to string on the kusto size? i guess you will another map for the string -> 2 to the power of x.
Please add a test for encoding and decoding so that i have an rough idea how it looks like in kusto.

Reply:
you might also consider when it's exceeding 64 bits, a new enum will be required? java has BitSet which can hold more than 64 bits.

Reply:
We have around 8 instrumentations currently so we must be good, I believe signed number used by bitwise operations in JS is only 32 bits